src/barebones_mpc/model/inv_pendulum_model.py

```diff
@@ -38,20 +61,21 @@ def predict_states(self, init_state, sample_input):
         """
         self.sample_states[0, :, :] = init_state
         for j in range(0, self.number_samples):
-            for i in range(1, self.sample_length+1):
-                self.sample_states[i, j, :] = self._predict(self.sample_states[i-1, j, :], sample_input[i-1, j, :])[:,0]
+            for i in range(1, self.sample_length + 1):
+                self.sample_states[i, j, :] = self._predict(self.sample_states[i - 1, j, :], sample_input[i - 1, j, :])[
+                    :, 0
+                ]
 
         return self.sample_states
```

Comment:
Here, predict_states returns sample_states, in the evaluator, there are methods to get_costs. Should we adopt a similar approach for all modules?